drivers/bms_ic/bq769x0/bq769x0.c

```diff
@@ -835,19 +923,28 @@ static void bq769x0_balancing_work_handler(struct k_work *work)
                 }
             }
 
-            LOG_DBG("Setting CELLBAL%d register to 0x%02X", section + 1, balancing_flags);
-
             ic_data->balancing_status |= balancing_flags << section * 5;
 
             /* set balancing register for this section */
-            bq769x0_write_byte(dev, BQ769X0_CELLBAL1 + section, balancing_flags);
+            err = bq769x0_write_byte(dev, BQ769X0_CELLBAL1 + section, balancing_flags);
+            if (err == 0) {
+                LOG_DBG("Set CELLBAL%d register to 0x%02X", section + 1, balancing_flags);
+            }
+            else {
+                LOG_ERR("Failed to set CELBAL%d register: %d", section + 1, err);
+            }
         }
     }
     else if (ic_data->balancing_status > 0) {
         /* clear all CELLBAL registers */
         for (int section = 0; section < dev_config->num_sections; section++) {
-            LOG_DBG("Clearing CELLBAL%d register", section + 1);
-            bq769x0_write_byte(dev, BQ769X0_CELLBAL1 + section, 0x0);
+            err = bq769x0_write_byte(dev, BQ769X0_CELLBAL1 + section, 0x0);
+            if (err == 0) {
+                LOG_DBG("Cleared CELLBAL%d register", section + 1);
+            }
+            else {
+                LOG_ERR("Clearing CELBAL%d register failed: %d", section + 1, err)
```

Comment:
I believe you would have spotted it, but there's a missing `;`, which is causing the build error.

Reply:
Thanks. Looks like I was too tired yesterday evening ;)

Fixed now.